docs/extend/modular-resources/create/create-subtype.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Define a New Resource Subtype"
+linkTitle: "New API Subtype"
+weight: 30
+type: "docs"
+tags: ["rdk", "extending viam", "modular resources", "API"]
+description: "Define an API for a new type of resource."
+no_list: true
+---
+
+You need to define a new resource *subtype* (and an API for that subtype) if:
+
+- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing [component](/components/) or [service](/services/) subtypes.
+- You have a resource that could fit into an existing subtype, but you want to define an API with different endpoints than the ones in the standard Viam [API](/program/apis/) for that subtype.
```

Comment:
```suggestion
- You have a resource that could fit into an existing subtype, but you want to define an API with different methods than the ones in the standard Viam [API](/program/apis/) for that subtype.
```

And should we say "standard VIam API" or "existing APIs" - as soon they could pick from APIs from third parties in the registry?

Reply:
That's a great point, but what's wrong with endpoints?

Reply:
```suggestion
- You have a resource that could fit into an existing subtype, but you want to define an API with different methods and messages than the ones in the existing [APIs](/program/apis/) for that subtype.
```